docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -0,0 +1,389 @@
+----
+
+. Ensure that you are using the service context that includes your Kafka and {registry} instances, as shown in the following example:
++
+[source,shell]
+----
+$ rhoas context use --name development-context
+----
+
+. In the guides and samples repository that you cloned, navigate to the directory for the Quarkus application.
++
+[source,shell]
+----
+$ cd ~/app-services-guides/code-examples/quarkus-service-registry-quickstart/
+----
+
+. Generate an environment variables file that contains the connection configuration information required by the producer component.
++
+[source,shell]
+----
+$ rhoas generate-config --type env --output-file ./producer/.env
+----
+
+. Copy the `.env` file to the directory for the consumer component, as shown in the following Linux example:
++
+[source,shell]
+----
+$ cp ./producer/.env ./consumer/.env
+----
++
+For a service context with single Kafka and {registry} instances, the `.env` file looks like the following example:
++
+.Example environment variables file for connection configuration information
+[source,shell]
+----
+## Generated by rhoas cli
+## Kafka Configuration
+KAFKA_HOST=kafka-inst-cafkr-jma--lhulbl-ca.bf2.kafka.rhcloud.com:443
+## Service Registry Configuration
+SERVICE_REGISTRY_URL=https://bu98.serviceregistry.rhcloud.com/t/0aa1dd8b-63d5-466c-9de8-7c03320a81c2
+SERVICE_REGISTRY_CORE_PATH=/apis/registry/v2
+SERVICE_REGISTRY_COMPAT_PATH=/apis/ccompat/v6
+
+## Authentication Configuration
+RHOAS_CLIENT_ID=srvc-acct-14295e3c-f72d-4bae-876c-3172a96eb7eb
+RHOAS_CLIENT_SECRET=5c3a20e0-d946-4edf-94d2-35db41f3b2ad
+RHOAS_OAUTH_TOKEN_URL=https://identity.api.openshift.com/auth/realms/rhoas/protocol/openid-connect/token
+----
++
+As shown in the example, the file that you generate contains the endpoints for your service instances, and the credentials required to connect to those instances. The CLI automatically created a service account (under the environment variable name `RHOAS_CLIENT_ID`) that client applications can use to authenticate with the Kafka and {registry} instances.
+
+. Set Access Control List (ACL) permissions to enable the new service account to access resources in the Kafka instance.
++
+.Example command for granting access to Kafka instance
+[source,shell]
+----
+$ rhoas kafka acl grant-access --producer --consumer --service-account srvc-acct-14295e3c-f72d-4bae-876c-3172a96eb7eb --topic quotes --group all
+----
++
+The command you entered allows applications to use the service account to produce and consume messages in the `quotes` topic. Applications can use any consumer group and producer.
+
+. Use Role-Based Access Control (RBAC) to enable the new service account to access the {registry} instance and the artifacts (such as schemas) that it contains.
++
+.Example command for granting access to {registry} instance
+[source,shell]
+----
+$ rhoas service-registry role add --role manager --service-account srvc-acct-14295e3c-f72d-4bae-876c-3172a96eb7eb
```

Comment:
This is fine, but the developer role might be more appropriate for a service account for this use-case.